README.md

```diff
@@ -556,16 +573,21 @@ metricExporter.Start();
 4. See [sample][ai-sample] for example use.
 
 ``` csharp
-using (var tracerFactory = TracerFactory.Create(builder => builder
-    .UseApplicationInsights(config => config.InstrumentationKey = "instrumentation-key")))
-{
-    var tracer = tracerFactory.GetTracer("application-insights-test");
-    var span = tracer
-        .SpanBuilder("incoming request")
-        .StartSpan();
+string FrontendKey = "my.org/keys/frontend";
+DistributedContext.Carrier = AsyncLocalDistributedContextCarrier.Instance; // Enable asynclocal carrier for the context

```

Comment:
Same as for Traces, I'm not sure we need to introduce / configure a DistributedContext in the example.

Reply:
should i remove?

Reply:
What do ou think about this:
```csharp
string FrontendKey = "my.org/keys/frontend";
using var tracerFactory = TracerFactory.Create(builder => builder
    .SetResource(Resources.CreateServiceResource("my-service"))
    .UseApplicationInsights(config => config.InstrumentationKey = "instrumentation-key"));
var tracer = tracerFactory.GetTracer("application-insights-test");

using (tracer.StartActiveSpan("incoming request", out var span))
{
    span.AddEvent("Start processing video.");
    Thread.Sleep(TimeSpan.FromMilliseconds(10));
    span.AddEvent("Finished processing video.");
}
```

Reply:
Yes, I think this is better. We can clear and concise examples.

Reply:
Ok! thanks for reviewing! just updated the samples! :)